policyengine_us/variables/gov/states/ny/nyserda/drive_clean/nyserda_drive_clean_vehicle_cost.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+class nyserda_drive_clean_vehicle_cost(Variable):
+    value_type = float
+    entity = Household
+    label = "Vehicle Price"
+    unit = USD
+    definition_period = YEAR
+    reference = https://www.nyserda.ny.gov/-/media/Project/Nyserda/Files/Programs/Drive-Clean-NY/implementation-manual.pdf#page=8
```

Comment:
```suggestion
    reference = "https://www.nyserda.ny.gov/-/media/Project/Nyserda/Files/Programs/Drive-Clean-NY/implementation-manual.pdf#page=8"
```

Reply:
we keep the quotation marks in the variable metadata but not in the parameter metadata

Reply:
Ah, gotcha! Sorry about that!